platforms/guardoni/.env.development

```diff
@@ -2,3 +2,5 @@ NODE_ENV=development
 YT_BACKEND=http://localhost:9000/api
 TK_BACKEND=http://localhost:14000/api
 DEBUG=@trex:*,guardoni:*
+PUBLIC_KEY=H7AsuUszehN4qKTj2GYYwNNzkJVqUQBRo2wgKevzeUwx
```

Comment:
@ascariandrea the right way to build a new keypair is to do `inspect background window` and copy if from a fresh profile, or there is any other way to generate it?

Reply:
@vecna I added a description to the PR - and the related issue